src/CMakeLists.txt

```diff
@@ -2088,6 +2094,24 @@ if(LINUX)
                 PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE
                             GROUP_READ GROUP_WRITE GROUP_EXECUTE
                             WORLD_READ             WORLD_EXECUTE)
+    elseif(EXISTS "/lib/x86_64-linux-gnu/libicui18n.so")
```

Comment:
would you consider using CMake's find module instead?
https://cmake.org/cmake/help/v3.15/module/FindICU.html

Reply:
I replaced the code with:
```
    INCLUDE(${CMAKE_ROOT}/Modules/FindICU.cmake)
    message("ICU_I18N_LIBRARY=${ICU_I18N_LIBRARY}")
    message("ICU_DATA_LIBRARY=${ICU_DATA_LIBRARY}")
    message("ICU_UC_LIBRARY=${ICU_UC_LIBRARY}")
    install(FILES ${ICU_I18N_LIBRARY} ${ICU_DATA_LIBRARY} ${ICU_UC_LIBRARY}
            DESTINATION ${VISIT_INSTALLED_VERSION_LIB}
            PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE
                        GROUP_READ GROUP_WRITE GROUP_EXECUTE
                        WORLD_READ             WORLD_EXECUTE)
```
and got the following output when I ran CMake on kickit.
```
-- Failed to find all ICU components (missing: ICU_LIBRARY) (found version "50.2")
ICU_I18N_LIBRARY=
ICU_DATA_LIBRARY=
ICU_UC_LIBRARY=
```
Is this the right approach? Any thoughts on getting this to work. I would of course make all the code after the include conditional on `ICU_FOUND`, but for debugging I left it out.

Reply:
Bummer. Perhaps this route is a bit too much work to get this icu library fix in for the release binaries where it is needed.  I was just concerned you would have to add yet another path in the future. I think a new ticket to investigate the more cmake-ified way of handling the problem is in order. Either with the FindICU as I had suggested, or find another way that wouldn't require path updates all the time.  Sorry for the extra work.

Reply:
I submitted issue #17868.